data_processors/pipeline/lambdas/gds_search.py

```diff
@@ -0,0 +1,88 @@
+try:
+    import unzip_requirements
+except ImportError:
+    pass
+
+import os
+import django
+import logging
+from data_processors.pipeline import services
+from libiap.openapi import libgds
+from utils import libjson, iap
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'data_portal.settings.base')
+django.setup()
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def get_presigned_url(file_id: str):
+    logger.debug(f"Retrieving pre-signed URL for file: {file_id}")
+    with libgds.ApiClient(iap.configuration(libgds)) as api_client:
+        file_api = libgds.FilesApi(api_client)
+    try:
+        file_details = libgds.FileListResponse = file_api.get_file(file_id=file_id)
```

Comment:
Ah huh. Found it! It should be `:` instead of `=` for Python type hinting. Strike to find that Python didn't fail it at runtime, as it is still a legit assignment syntax; it does assign `libgds.FileListResponse` value from right side call; that's effectively overwrite `libgds` module throughout the app that point onwards!! Yike, downside of Python duck typing, I suppose!! I will push improvement/fix in tick, next.

```
file_details: libgds.FileListResponse = file_api.get_file(file_id=file_id)
```

Reply:
Ah, thanks! 
Didn't spot this....